eslint.config.js

```diff
@@ -74,7 +74,8 @@ module.exports = [
             "tests/fixtures/**",
             "tests/performance/**",
             "tmp/**",
-            "**/test.js"
+            "**/test.js",
+            "**/svgo.config.*js"
```

Comment:
Why ignore this file? Wouldn't we want it to be linted with the same standards as other config files?